rest_docs/rest-catalog-open-api-v0.1.yaml

```diff
@@ -0,0 +1,727 @@
+          description: Additional catalog level properties
+          default: {}
+      required:
+        - name
+        - properties
+    CreateNamespaceRequest:
+      type: object
+      properties:
+        namespace:
+          type: array
+          description: individual levels of the namespace
+          items:
+            type: string
+        properties:
+          type: object
+          description: Configuration properties for the namespace
+          example: '{ owner: "Hank Bendickson" }'
+    RenameTableRequest:
+      type: object
+      properties:
+        sourceTableIdentifier:
+          $ref: '#/components/schemas/TableIdentifier'
+        destinationTableIdentifier:
+          $ref: '#/components/schemas/TableIdentifier'
+    CreateTableRequest:
+      type: object
+      properties:
+        identifier:
+          $ref: '#/components/schemas/TableIdentifier'
+        schema:
+          $ref: '#/components/schemas/Schema'
+        partitionSpec:
+          $ref: '#/components/schemas/PartitionSpec'
+        sortOrder:
+          $ref: '#/components/schemas/SortOrder'
+        properties:
+          type: object
+          additionalProperties:
+            type: string
+        metadataJson:
+          type: string
+        commit:
+          type: boolean
+    PartitionSpec:
+      type: object
+      properties:
+        unpartitioned:
+          type: boolean
+    Schema:
+      type: object
+      properties:
+        aliases:
+          type: object
+          additionalProperties:
+            type: integer
+            format: int32
+    SortOrder:
+      type: object
+      properties:
+        unsorted:
+          type: boolean
+    CreateTableResponse:
+      type: object
+      properties:
+        identifier:
+          $ref: '#/components/schemas/TableIdentifier'
+        tableLocation:
+          type: string
+          description: "Path of of the table just before the standard \
+            metadata / data folders. This is useful in cases where a different location
+            provider might be used"
+          example: s3://bucket/prod/accounting.db/monthly_sales
+        metadataLocation:
+          type: string
+          description: "Location of the most current primary metadata file for the table"
+          example: s3://bucket/prod/accounting.db/monthly_sales/metadata/00001-0d4ef14f-ef7d-43e7-9af-5f4ad40a1103.metadata.json
+        metadataJson:
+          type: string
+          description: "Stringified JSON representing the tables metadata"
+          example:
+            "TODO"
+        accessToken:
+          type: string
+    SetPropertiesRequest:
+      type: object
+      properties:
+        namespace:
+          type: string
+        properties:
+          type: object
+    ListNamespacesResponse:
+      type: object
+      properties:
+        databases:
+          type: array
+          items:
+            $ref: '#/components/schemas/Namespace'
+    Namespace:
```

Comment:
`NamespaceProperties`? It may make sense to define `Namespace` as a list of strings. Or maybe `NamespaceIdentifier`.